src/onnxruntime.cc

```diff
@@ -1283,6 +1310,39 @@ ModelInstanceState::SetInputTensors(
     }
   }
 
+  // Process batch input if any
```

Comment:
Indent? Note that we have a formatting tool that you can run that should normalize all code formatting. https://github.com/triton-inference-server/server/blob/master/CONTRIBUTING.md#coding-convention